chat.go

```diff
@@ -13,6 +13,25 @@ import (
 // about a chat command that's taking long to execute.
 var ChatCmdTimeout = 10 * time.Second
 
+// Exec handles a string as if it where send by a client.
+// string is imediate returnvalue of chatcommand (if applicable)
+// bool   is if msg is getting forwarded to the minetest server
+func (cc *ClientConn) Exec(msg string) (string, bool) {
+	pkt := &mt.ToSrvChatMsg{
```

Comment:
You can actually just write &mt.ToSrvChatMsg{msg} here

Reply:
no, you actually cant:
```
# github.com/HimbeerserverDE/mt-multiserver-proxy
./chat.go:18:10: github.com/anon55555/mt.ToSrvChatMsg composite literal uses unkeyed fields
```

Reply:
Interesting, mt magic